product-service/src/books/books.router.js

```diff
@@ -0,0 +1,24 @@
+  
+const router = require('express').Router();
```

Comment:
What was the reason to use the 'express' ? because at the meeting we discussed that it is better to avoid it

Reply:
I missed that thing at the meeting. Discussed that thing with Sergey Kovalchuk1. He said that it's bad practice. And that I will rewrite code wiyhout express for the nex 4 task :)